libaleth-interpreter/VMConfig.h

```diff
@@ -0,0 +1,433 @@
+/*
```

Comment:
Hm, why does github doesn't show this file as moved

Reply:
Did you change it?

Reply:
Ah is it copied?

Reply:
It's a copy. The config was shared between legacy and interpreter VMs.

Reply:
Perhaps you want to move the file to `libaleth-interpreter` and place a copy into the original directory?

That way if `libaleth-interpreter` is split out of the main tree using "git magic" the history will stay proper.

Reply:
I don't think VMConfig is super important, but I can try to do it.